Sources/Target/Windows/Process.cpp

```diff
@@ -350,6 +351,22 @@ ErrorCode Process::writeMemory(Address const &address, void const *data,
   return kSuccess;
 }
 
+ErrorCode Process::getFileLoadAddress(std::string const &file_path,
```

Comment:
Seems like this function doesn't do anything windows-specific. Maybe you can just pull it to `ProcessBase` as is?